docker-build/Dockerfile

```diff
@@ -7,4 +7,4 @@ WORKDIR /palladium-view
 RUN npm install -g @angular/cli --unsafe-perm && npm install
 
 ENTRYPOINT ["ng"]
-CMD ["build", "--prod", "--aot=false"]
+CMD ["build", "--aot=false", "--configuration production"]
```

Comment:
Can this be repalced with `CMD ["nb", "build"]` since all options already defined in `package.json` with same keys?

Reply:
I think it's better to run the build via `npm run` in containers so as not to drag an additional global installation of the angular cli

```bash
npm install -g @angular/cli --unsafe-perm
```